restructure release health page (#5032)
getsentry/sentry-docs: src/docs/product/releases/health/index.mdx

---
title: Release Health
sidebar_order: 50
redirect_from:
  - /workflow/releases/health/
description: "Monitor the health of releases by observing user adoption, usage of the application, percentage of crashes, and session data."
---

> You can monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session). Release health data provides insight into the impact of crashes and bugs related to user experience and reveals trends with each new issue through the release details, graphs, and filters on the. This data is primarily displayed on the [Releases](/product/releases/) and [Release Details](/product/releases/release-details) pages, but is also reflected in other parts of [sentry.io](https://sentry.io).

Author: ```suggestion
You can monitor the health of releases by observing user adoption, usage of your application, percentage of [crashes](#crash), and [session data](#session). Release health data provides insight into the impact of crashes and bugs related to user experience and reveals trends with each new issue through the release details, graphs, and filters on the. This data is primarily displayed on the [Releases](/product/releases/) and [Release Details](/product/releases/release-details) pages, but is also reflected in other parts of [sentry.io](https://sentry.io).
```

> Many SDKs automatically manage the start and end of [sessions](#session) when the SDK is initialized, but release health configuration is key to ensuring you're receiving useful data, so check out our information on [which SDKs support release health](/product/releases/health/setup/) for links to configuring the SDK, as needed.

Once you configure your SDK, Sentry connects the data to the specific release of your application and the associated code. This page describes some key release health concepts and what you can learn from each type of release health data.

> ## Sessions Data {#session}

Author: ```suggestion
## Sessions {#session}
```

### Active Sessions/Users

> "Active Sessions" are sessions that started in the application in the last 24 hours.

Author: ```suggestion
_Active sessions_ are sessions that started in the application in the last 24 hours.
```

"Active Users" are users who started the application at least once in the last 24 hours.

> Depending on your "Display" selection, the sort options for the page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".

Author: ```suggestion
Depending on your "Display" selection, the sort options for the **Releases** page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".
```

### Active Sessions/Users

_Active sessions_ are sessions that started in the application in the last 24 hours.

_Active users_ are users who started the application at least once in the last 24 hours.

Depending on your "Display" selection, the sort options for the **Releases** page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".

![Sort By: Active Sessions dropdown](release_active_sessions.png)

![Sort By: Active Users dropdown](release_active_users.png)

> ## Crash Data {#crash}

Author: ```suggestion
## Crashes {#crash}
```

A _crash_ is when the application had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue that captures this event in [sentry.io](https://sentry.io), and errors that did not cause the end of the application should not be included. To search for crash events in **Discover** or on the **Issues** page, filter by `error.unhandled:true`. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [inbound filters](/product/data-management-settings/filtering/) or [sampling](/platform-redirect/?next=/configuration/sampling/).

### Crash Free Sessions/Users

The _crash free sessions_ number is the percentage of sessions in the specified time range not ended by a crash of the application.

> The _crash free users_ number is the percentage of distinct users who did not experience a crash during the specified time period.

Reviewer: tempted to say something about alerts here

> The _crash free users_ number is the percentage of distinct users who did not experience a crash during the specified time period.

Author: ```suggestion
The _crash free users_ number is the percentage of distinct users who did not experience a crash during the specified time period. 

You can set [crash rate alerts](/product/alerts/alert-types/#sessions-crash-rate-alerts) to tell you when your crash free percentage for either sessions or users falls below a specific threshold. 
```

Depending on your "Display" selection, the sort options for the **Releases** page change. To sort releases by crash free sessions, select "Sessions" in the "Display" dropdown, and to sort by crash free users, select "Users".

![Sort By: Crash Free Sessions dropdown](release_crash_free_sessions.png)

![Sort By: Crash Free Users dropdown](release_crash_free_users.png)

### Crashed Users

The number of users that experienced a crash in the specified time range.

> ## Release Adoption Data {#release-adoption}

Author: ```suggestion
## Release Adoption
```

### Crashed Users

The number of users that experienced a crash in the specified time range.

## Release Adoption Data {#release-adoption}

Depending on the selected display option, we show either:

- Session adoption - The number of sessions of a specific release in the last 24 hours.
- User-based adoption - The number of users who started the application at least once during the last 24 hours of a specific release.

> The adoption progress bar is calculated as a percentage of sessions/users in this release vs all releases in the project, in the last 24 hours:

Author: ```suggestion
The adoption progress bar is calculated as a percentage of sessions/users in this release compared to all releases in the project, in the last 24 hours:
```

![Adoption stage label on the Releases page.](adoption-stage-label.png)

<Note>

Adoption stage labels are only displayed for mobile projects when you filter down to a single environment.

</Note>

For each release, the release's percentage of sessions is calculated against all sessions in the same environment and project. It's then marked as being in one of three stages, based on a threshold of 10%. These stages are:

- Adopted - The release's percentage of sessions compared to all sessions is above the threshold. That is, the release was seen in 10% or more of sessions over all releases in the last six hours.
- Low Adoption - The release's percentage of sessions compared to all sessions that are below the threshold. That is, the release was seen in less than 10% of sessions over all the releases in the last six hours.
- Replaced - The release was previously tagged as "Adopted", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "Adopted".

> ## Release Status Data

Author: ```suggestion
## Session Status Data
```

> ## Session Status Data

Author: ```suggestion
## Session Status
```

A session can have one of the following statuses:

### Abnormal

> The application timed out, froze, or was forced to quit by the operating system. There is usually no corresponding Sentry issue, as this is a passive action.

Author: ```suggestion
The application timed out, froze, or was forced to quit by the operating system. There is usually no corresponding Sentry issue, as this is a passive action.

### Crashed

The application had an explicit unhandled error or hard [crash](#crash).
```
